### DISCUSSION TITLE
Consider change some `const string` to `static readonly string`

### DISCUSSION THREAD

**Discussion by Cologler** [devlooped/ThisAssembly#63] [Ideas]
Based on some articles:

- https://stackoverflow.com/questions/755685/static-readonly-vs-const
- https://stackoverflow.com/questions/3190870/const-string-vs-static-readonly-string-in-c-sharp

If any package has not to recompile, we may get the wrong information. For example, if the main process tries to read the version string from a hotfix dll file, it will always read the wrong value.

**Comment by kzu** [comment]
Yep, sounds reasonable. Created https://github.com/devlooped/ThisAssembly/issues/64 to track its implementation. 

Thanks!
